response-datastore/response-server-service/src/main/resources/application.properties

```diff
@@ -13,8 +13,8 @@ spring.main.allow-bean-definition-overriding=true
 #logging.file=${logging.path}/response_datastore_service.log
 #logging.file.max-size=1024KB
 
-logging.level.root=WARN
-logging.level.com.google.cloud.healthcare.fdamystudies=WARN
+logging.level.root=DEBUG
```

Comment:
Created Issue #2851 to revert the log level to WARN after troubleshooting the issue.